client/src/components/LikesList.vue

```diff
@@ -0,0 +1,125 @@
+<template>
+  <div>
+    <q-dialog v-model="info.show" position="bottom">
+      <q-card>
+        <q-bar class="dialog-header text-white">
+          <div class="text-h6">Likes</div>
+          <q-space />
+
+          <q-btn dense flat icon="close" v-close-popup>
+            <q-tooltip>Close</q-tooltip>
+          </q-btn>
+        </q-bar>
+
+        <q-list>
+          <q-item v-for="like in likes" :key="like.id" class="q-my-sm" clickable>
+            <q-item-section avatar>
+              <q-avatar color="primary" text-color="white">
+                <img :src="`https://cdn.quasar.dev/img/${like.avatar}`" />
+              </q-avatar>
+            </q-item-section>
+
+            <q-item-section>
+              <q-item-label>{{ like.name }}</q-item-label>
+            </q-item-section>
+            <q-item-section side>
+              <q-item-label caption>{{ like.date | howMatchTimeAgo}}</q-item-label>
+            </q-item-section>
+
+            <q-item-section side>
+              <i class="fas fa-heart"></i>
+            </q-item-section>
+          </q-item>
+        </q-list>
+
+        <q-card-actions>
+          <q-btn label="CLOSE LIST" style="background:#4db6ac;" color="#fff" class="full-width" v-close-popup />
+        </q-card-actions>
+      </q-card>
+    </q-dialog>
+  </div>
+</template>
+
+<script>
+const likes = [{
+  id: 1,
+  name: 'Ruddy Jedrzej',
+  date: '1570062998222',
+  avatar: 'avatar1.jpg'
+}, {
+  id: 2,
+  name: 'Jan Khalib',
+  date: '1572273790143',
+  avatar: 'avatar2.jpg'
+}, {
+  id: 3,
+  name: 'Gorge Lebiskiy',
+  date: '1574291790682',
+  avatar: 'avatar4.jpg'
+}, {
+  id: 4,
+  name: 'Dayana Fawdrey',
+  date: '1574073740192',
+  avatar: 'avatar3.jpg'
+}, {
+  id: 5,
+  name: 'Mallorie Zuck',
+  date: '1574291846223',
+  avatar: 'avatar2.jpg'
+}, {
+  id: 6,
+  name: 'Mallorie Zuck',
+  date: '1575762998222',
+  avatar: 'avatar2.jpg'
+}, {
+  id: 7,
+  name: 'Gorge Lebiskiy',
+  date: '1575062998222',
+  avatar: 'avatar4.jpg'
+}
+]
+export default {
+  props: ["info"],
+  data () {
+    return {
+      likes
+    }
+  },
+  methods: {
+  },
+  mounted () {
+    likes.sort((a, b) => {
+      return b.date - a.date
+    })
+  },
+  filters: {
+    howMatchTimeAgo: function (date) {
```

Comment:
This is "developer's nightmare" - please don't reinvent the wheel.
Moment.js has "moment(new Date(timeStamp)).fromNow()" functionality.

On real projects you will be forced to do translations of such things yourself unless library has it covered (and moment has all math and translations in cases of importing and properly switching them)